src/screens/LessonsList.js

```diff
@@ -13,6 +13,19 @@ const lessonsList = {
 const LessonsList = ({route, navigation, theme}) => {
   const {levelNumber} = route.params;
 
+  const renderSeperator = () => {
+    return (
```

Comment:
if you're only returning do insteead

`const renderSeperator = () => <View /> `

basically remove the `{ } ` and the `return`